shopinvader_v1_base/migrations/14.0.5.14.1/post-migrate.py

```diff
@@ -9,8 +9,8 @@ def _upgrade_shopinvader_model_access(env):
     Upgrade the "shopinvader partner binding edit" access rights
```

Comment:
TODO
- not sure that the migration script should be moved, if you have the module installed, I thinks it should stay in shopinvader module